Release/src/http/client/http_client_asio.cpp

```diff
@@ -1046,7 +1050,18 @@ class asio_context : public request_context, public std::enable_shared_from_this
             }
             else
             {
-                report_error("Failed to read HTTP status line", ec, httpclient_errorcode_context::readheader);
+                std::string error_msg;
+                std::string category(ec.category().name());
+                if (ec.value() == 125)
+                {
+                    error_msg = "Failed to read HTTP status line due to http_client object reaching maximum timeout.: error category: " + category + " error message: " + ec.message() + " error value: " + std::to_string(ec.value());
+                    std::cout << error_msg << std::endl;
```

Comment:
> std::cout << error_msg << std::endl; [](start = 19, length = 37)

you probably added this std::cout is for debugging purpose could you remove it before check-in.

Reply:
fixed.